### DISCUSSION TITLE
Updating a ProgressBar using Update-BTNotification

### DISCUSSION THREAD

**Discussion by Negwoh** [Windos/BurntToast#109] [General]
I can't get this to work, the cmdlet has no -ProgessBar param and databinding only works for strings, can someone explain to me how I can update a progress bar using this cmdlet (or any other method other than new-burnttoastnotification as this is obtuse to the user :)).

Thanks in advance!

**Comment by Windos** [comment]
Hey @Negwoh, you need to dive into update-able notifications (as it looks like you've started down).

As luck would have it, those databindings can be used for more than strings, and especially those progressbars.

For general overview of update-able toast, checkout this [post](https://toastit.dev/2019/07/17/crouton-11/).

But for something more specific to progress bars (and databindings with more than strings), checkout this reply to another [discussion](https://github.com/Windos/BurntToast/discussions/103#discussioncomment-36394).

**Reply by Negwoh** [reply]
Thanks for the quick response here, I actually broke my own rule and didn't check the other discussions before posting, only the issues, and have subsequently found that post. I will give this a go and report back!

PS: Great to see a kiwi producing high quality, useful PS modules!

**Reply by Windos** [reply]
No prob. The addition of discussions on GitHub is going to be one of those things to get used to... it's a whole nother place to check for past chatter.

The good news is I've had some common question lately which helps steer me towards which documentation I need to fix (because it is deficient in a lot of places) first.

**Comment by jfthuong** [comment]
@Negwoh You can probably mark the topic as "Answered" :)

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
